Pod/Classes/TMPodspec.m

```diff
@@ -61,21 +76,125 @@ - (NSString *)version {
     return [self stringForKey:PodspecKeyVersion];
 }
 
-- (NSString *)summary {
-    return [self stringForKey:PodspecKeySummary];
+- (NSArray *)authors {
+    NSString *authorName = [self stringForKey:PodspecKeyAuthor];
+    NSArray *authorNames = [self arrayForKey:PodspecKeyAuthors];
+    NSDictionary *authorNamesAndAddresses = [self dictionaryForKey:PodspecKeyAuthors];
```

Comment:
Yeah just extra stuff in scope. It's fine though, more just me thinking out loud than anything else.